amalgamate.mjs

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env node
```

Comment:
In the future, could we use either Python or shell scripts?

Reply:
Yeah.  Shell probably isn't a good idea; it's too complex.  But we could do Python.  It started out as some very basic scripting, and since `compile_commands.json` is JSON, I used the `node` REPL, and then it grew, but I only worked on it for a couple of days quickly and dirtily, so it wasn't worth rewriting it much.  And for `cfs-rust`, I just copied the one from `lighttpd-rust` and modified it.